openwebstart/src/test/java/com/openwebstart/util/FilenameUtilTest.java

```diff
@@ -0,0 +1,55 @@
+package com.openwebstart.util;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+class FilenameUtilTest {
+
+    @ParameterizedTest
+    @MethodSource("getNamesThatShouldNotChange")
+    public void testValidFileNameWithNormalName(String data) {
+        //given
+        final String name = data;
+
+        //when
+        final String converted = FilenameUtil.toSimplifiedFileName(name);
+
+        //than
```

Comment:
than -> then

Reply:
oh, shit. I created this branch not from the master :D Should only be 1 file that has changed... Will redo